lunch-go-where/src/pages/Landing.js

```diff
@@ -52,29 +55,51 @@ const Landing = () => {
     setNewUsername(evt.target.value);
   };
   const handleNewPasswordChange = (evt) => {
-    setNewPassword(evt.target.value);
+    const testPassword = evt.target.value;
+    const condition = /^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,}$/;
+    const test = condition.test(testPassword);
+    if (test) {
+      setNewPassword(testPassword);
+      setIsValid(true);
+    } else {
+      console.log("Password Fail");
+    }
+    setNewPassword(testPassword);
   };
   const handleSubmit = async (evt) => {
     evt.preventDefault();
     console.log("SUBMITTING", username, password);
-    await apis.login({ username: username, password: password });
-  };
-  const handleSignUpClick = async (evt) => {
-    evt.preventDefault();
     try {
-      const res = await apis.postSignUp({
-        name: name,
-        username: newUsername,
-        password: newPassword,
-      });
-      setName("");
-      setNewUsername("");
-      setNewPassword("");
+      await apis.login({ username: username, password: password });
+      setPassword("");
```

Comment:
here it's good to reset the name and uesrname to "" as well?

Reply:
Yes sorry i miss out for this have added it in